src/components/connection_handler/test/connection_handler_impl_test.cc

```diff
@@ -412,13 +414,13 @@ TEST_F(ConnectionHandlerTest, OnMalformedMessageCallback) {
 
   InSequence seq;
   EXPECT_CALL(mock_connection_handler_observer,
-              OnServiceEndedCallback(connection_key_,kMobileNav, kMalformed)).Times(0);
+              OnServiceEndedCallback(connection_key_,kRpc, kMalformed)).Times(1);
```

Comment:
@pestOO , Changed const iterator to const reverse interator, fixed test, [new pull request] (https://github.com/smartdevicelink/sdl_core/pull/325)